PANCHANGE_data_preprocessing2020-07-15.Rmd

```diff
@@ -618,28 +618,32 @@ dem.raw.id %>%
 ## Read in data
 ```{r ramp emp cols}
 if(RAMP == TRUE){
-  ramp.emp.raw <- readRDS(file = paste0(data.raw_path, "/ramp/employment_ramp.rds"))
-  dim(ramp.emp.raw)
-  colnames(ramp.emp.raw)
+  ramp.employment.raw <- readRDS(file = paste0(data.raw_path, "/ramp/employment_ramp.rds"))
+  dim(ramp.employment.raw)
+  colnames(ramp.employment.raw)
 }
 ```
 
 # Select variables
 ```{r ramp emp cleaning}
 if(RAMP == TRUE){
-  exclude_cols_emp <- c("ID",
-                        "Sample")
+  exclude_cols_employment <- c("ID",
+                        "Sample",
+                        "startDate.ramp")
```

Comment:
It's not startDate.ramp, it must be startDate.coping otherwise the dates are not going to be in the same column